### PR TITLE
Feat/create add game page

### PR DESCRIPTION
**Pull request by enricodelarosa**
## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by idiglove — COMMENTED** [review]
this looks like PR #11  too?

**Comment by enricodelarosa** [comment]
> this looks like PR #11 too?



> this looks like PR #11 too?

Yes, but with 2 more commits :)

File: project_app/pages/addgame.tsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+export default function admin() {
+    const [gameTitle, setGameTitle] = useState("");
+    const [gameDesc, setGameDesc] = useState("");
+
+    async function handleAddGame() {
+/*         console.log('clicked hangle game')
+        console.log({
+            gameTitle: gameTitle,
+            gameDesc: gameDesc
+        }) */
```
Comment (idiglove) [inline project_app/pages/addgame.tsx:21]: remove comments pls

File: project_app/pages/addgame.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  font-family: Rubik, sans-serif;
+border-radius: 0.625rem;
+font-size: 1.25rem;
+  min-width: 10rem;
+padding: 0.5rem;
+outline: none;
+border: none;
+cursor: pointer;
+text-transform: capitalize;
+`;
```
Comment (idiglove) [inline project_app/pages/addgame.tsx:205]: put this in a separate file thanks

**Review by idiglove — CHANGES_REQUESTED** [review]

**Pull request closed without merging**